spec/support/certificate_support.rb

```diff
@@ -3,7 +3,7 @@ def generate_cert(**args)
     generate_rsa_cert_and_key(args)[0]
   end
 
-  def generate_rsa_cert_and_key(expires_in: 1.year, size: 2048, cn: "GENERATED TEST CERTIFICATE")
+  def generate_rsa_cert_and_key(expires_in: 1.year, size: 2048, cn: "GENERATED TEST CERTIFICATE", digest: nil)
```

Comment:
why the digest here?

Reply:
Because as the args are passed through... Having the digest here stops it from throwing an error because digest is present.